app/models/user.rb

```diff
@@ -3,7 +3,16 @@
 class User < ActiveRecord::Base
   has_secure_password 
   has_many :reviews
+  has_many :queue_items
 
   validates :email, presence: true, uniqueness: true
   validates :full_name, presence: true
+
+  def has_video_in_queue?(video)
+    !!video_queue_item(video)
```

Comment:
You can refactor this - `queue_items.map(&:video).include? video`. What do you think?